packages/scripts/config/webpack.config.js

```diff
@@ -66,6 +66,8 @@ const externals = [
 const isProduction = process.env.NODE_ENV === 'production';
 const mode = isProduction ? 'production' : 'development';
 
+const getBabelLoaderOptions = () => hasBabelConfig() ? {} : { presets: [ require.resolve( '@wordpress/babel-preset-default' ) ] };
```

Comment:
I figured out we should ensure it doesn't load any random configs when we provide our own. `react-scripts` does it as follows:

https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/config/webpack.config.js#L354-L355

Reply:
Done at 06f132b

It shouldn't pick anything because we check for the babel config first. Yet, I've tested that `hasBabelConfig` is stricter than babel itself (a file named `.babelrc.js` won't be detected as babel config by `hasBabelConfig` by it will by babel). So: added those checks to be future-proof.

Also added `.babelrc.js` as a valid babel config in dbbc04b.